engo.go

```diff
@@ -5,14 +5,20 @@ import (
 	"log"
 
 	"engo.io/ecs"
+	"engo.io/engo/act"
 )
 
 var (
 	// Time is the active FPS counter
 	Time *Clock
 
 	// Input handles all input: mouse, keyboard and touch
-	Input *InputManager
+	Input *InputMgr
```

Comment:
Well, I don't agree. I'm against things like variable names as `o` or `s` because then I have to search for signatures or whatever. Even when I'm the guy who did the code. I always prefer things a little more verbose but easier to understand what they are.

Anyway, just my humble opinion!